compiler/pragmas.nim

```diff
@@ -158,6 +158,12 @@ proc processImportCpp(c: PContext; s: PSym, extname: string, info: TLineInfo) =
     incl(m.flags, sfCompileToCpp)
   incl c.config.globalOptions, optMixedMode
 
+proc processImportJs(c: PContext; s: PSym, pattern: string, info: TLineInfo) =
+  setExternName(c, s, pattern, info)
+  incl(s.flags, sfImportc)
```

Comment:
``importjs`` should also support patterns! You need to add ``incl(s.flags, sfInfixCall)`` for this. And maybe a test.